ios/RNSInvalidateControllerProtocol.h

```diff
@@ -0,0 +1,5 @@
+#import <Foundation/Foundation.h>
+
+@protocol RNSInvalidateControllerProtocol <NSObject>
```

Comment:
```suggestion
@protocol RNSInvalidateControllerProtocol
```
Do we need NSObject here?

Reply:
```suggestion
@protocol RNSViewControllerInvalidating
```

I think that we're trying to follow Swift naming convention (https://www.swift.org/documentation/api-design-guidelines/#naming).

> Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).